src/components/Vuetify/index.js

```diff
@@ -9,18 +11,21 @@ const Vuetify = {
 
     this.installed = true
 
-    const $vuetify = {}
-    Vue.util.defineReactive($vuetify, 'inspire', {
-      breakpoint: {},
-      application,
-      dark: false,
-      theme: theme(opts.theme),
-      options: options(opts.options),
-      goTo
+    process.env.NODE_ENV !== 'test' && checkVueVersion(Vue)
```

Comment:
Why shouldn't we check vue version in test env?

Reply:
Because the tests don't go through webpack, so each test dumps "Vuetify requires Vue version undefined" to the console.